test/dataclasses/test_byte_stream.py

```diff
@@ -35,6 +37,25 @@ def test_from_string():
     assert b.meta == {"foo": "bar"}
 
 
+def test_to_string():
+    test_string = "Hello, world!"
+    b = ByteStream.from_string(test_string)
+    assert b.to_string() == test_string
+
+
+def test_to_string_encoding():
```

Comment:
Since `utf-8` is the default, I would use a different encoding in this test

Reply:
@masci something like [dd4d08c](https://github.com/deepset-ai/haystack/pull/7009/commits/dd4d08c0bacb819c69492de7d1474c7d705faac0) ?